src/app/modules/admin/state/admin.effects.ts

```diff
@@ -0,0 +1,24 @@
+import { Injectable } from '@angular/core';
+import { Actions, createEffect, ofType } from '@ngrx/effects';
+import { of } from 'rxjs';
+import { switchMap } from 'rxjs/operators';
+import * as AdminActions from './admin.actions';
+
+@Injectable()
+export class AdminEffects {
+    effectName$ = createEffect(() => {
+        return this.actions$.pipe(
+            ofType(AdminActions.loadAdminInfo),
+            switchMap(() =>
+                of({
+                    type: null,
+                    name: 'Seva',
+                    email: 'seva@exade.com',
+                    avatar: 'asdasdasdasdasd',
+                }),
+            ),
```

Comment:
oh, i forgot to told you one important thing here...
rewrite it in this way:
 getIsVoted$ = createEffect(() => this.actions$.pipe(
        ofType(IsVotedActions.getIsVoted),
        switchMap(_ => this.scoringService.getShowProjectSelection()
            .pipe(
                map(({isShowProjectSelection}) => {
                    return IsVotedActions.getIsVotedSuccess({ isVoted: isShowProjectSelection });
                }),
            )),
    ));
    
   so you should create loadAdminInfoSucces event and then in reducer you should wait this success action and put data to the store